src/core/hle/ipc.h

```diff
@@ -56,8 +56,8 @@ union Header {
  * @note While @p normal_params_size is equivalent to the number of normal parameters,
  * @p translate_params_size includes the size occupied by the translate parameters headers.
  */
-inline u32 MakeHeader(u16 command_id, unsigned int normal_params_size,
-                      unsigned int translate_params_size) {
+inline constexpr u32 MakeHeader(u16 command_id, unsigned int normal_params_size,
+                                unsigned int translate_params_size) {
```

Comment:
```suggestion
inline constexpr u32 MakeHeader(u16 command_id, u32 normal_params_size,
                                u32 translate_params_size) {
```
can be updated while we are at it